services/121-service/src/payments/payments.service.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { TestBed } from '@automock/jest';
+import { Repository } from 'typeorm';
+
+import { PaymentsService } from '@121-service/src/payments/payments.service';
+import { TransactionStatusEnum } from '@121-service/src/payments/transactions/enums/transaction-status.enum';
+import { TransactionScopedRepository } from '@121-service/src/payments/transactions/transaction.repository';
+import { ProgramEntity } from '@121-service/src/programs/program.entity';
+import { RegistrationStatusEnum } from '@121-service/src/registration/enum/registration-status.enum';
+import { RegistrationScopedRepository } from '@121-service/src/registration/repositories/registration-scoped.repository';
+
+function createMockTransaction(
+  id: number,
+  amount: number,
+  status: TransactionStatusEnum,
+) {
+  return {
+    id,
+    registrationId: id,
+    registrationStatus: RegistrationStatusEnum.included,
+    amount,
+    status,
+    created: new Date(),
+    updated: new Date(),
+    payment: 1,
+    registrationProgramId: id + 100,
+    referenceId: `REF-${id}`,
+    errorMessage: null,
+    programFinancialServiceProviderConfigurationName: 'someFsp',
+  };
+}
+
+const mockTransactions = [
+  createMockTransaction(101, 100, TransactionStatusEnum.success),
+  createMockTransaction(102, 200, TransactionStatusEnum.success),
+];
+
+describe('PaymentsService - getTransactions', () => {
+  let service: PaymentsService;
+  let transactionScopedRepository: TransactionScopedRepository;
+  let programRepository: Repository<ProgramEntity>;
+  let registrationScopedRepository: RegistrationScopedRepository;
+
+  const mockFullnameNamingConvention = ['firstName', 'lastName'];
```

Comment:
Why mock two name fields when all projects have it in 1 field according to requirements?